third_party/terraform/resources/resource_compute_network_peering.go.erb

```diff
@@ -65,9 +80,13 @@ func resourceComputeNetworkPeeringCreate(d *schema.ResourceData, meta interface{
 	}
 
 	request := &compute.NetworksAddPeeringRequest{
-		Name:             d.Get("name").(string),
-		PeerNetwork:      d.Get("peer_network").(string),
-		AutoCreateRoutes: d.Get("auto_create_routes").(bool),
+		Name:             	d.Get("name").(string),
+		PeerNetwork:      	d.Get("peer_network").(string),
+		AutoCreateRoutes: 	d.Get("auto_create_routes").(bool),
+		<% unless version == 'ga' -%>
+		ImportCustomRoutes:	d.Get("import_custom_routes").(bool),
```

Comment:
These fields have been deprecated and now appear in the nested `NetworkPeering` object.